src/Object/Contact.php

```diff
@@ -386,12 +376,12 @@ public static function photoMenu(array $contact, $uid = 0)
 		if ($contact['uid'] != $uid) {
 			if ($uid == 0) {
 				$profile_link = zrl($contact['url']);
-				$menu = Array('profile' => array(t('View Profile'), $profile_link, true));
+				$menu = array('profile' => array(t('View Profile'), $profile_link, true));
 
 				return $menu;
 			}
 
-			$r = q("SELECT * FROM `contact` WHERE `nurl` = '%s' AND `network` = '%s' AND `uid` = %d", dbesc($contact['nurl']), dbesc($contact['network']), intval($uid));
+			$r = dba::select('contact', array(), array('nurl' => $contact['nurl'], 'network' => $contact['network'], 'uid' => $uid));
 			if ($r) {
 				return self::photoMenu($r[0], $uid);
```

Comment:
Either use `limit => 1` or `dba::fetch($r)`, `$r[0]` is invalid with `dba::select()`